presto-main/src/main/java/com/facebook/presto/sql/planner/LocalExecutionPlanner.java

```diff
@@ -597,6 +600,40 @@ public PhysicalOperation visitRemoteSource(RemoteSourceNode node, LocalExecution
         }
 
         @Override
+        public PhysicalOperation visitMerge(MergeNode node, LocalExecutionPlanContext context)
+        {
+            List<Type> types = getSourceOperatorTypes(node, context.getTypes());
+
+            // Enforce a driver instance count of 1 since we use multiple exchange clients
+            context.setDriverInstanceCount(1);
+
+            Map<Symbol, Integer> sourceLayout = makeLayoutFromOutputSymbols(node.getOutputSymbols());
+            List<Symbol> orderBySymbols = node.getOrderBy();
+            List<Integer> sortChannels = getChannelsForSymbols(orderBySymbols, sourceLayout);
+            List<SortOrder> sortOrder = orderBySymbols.stream()
```

Comment:
orderings is ` Map<Symbol, SortOrder>` -- you need to get the `sortOrder` for each of the specified `orderBySymbols` in the right order.